test/fixtures/requests/prepare-settings-no-signer-entries.json

```diff
@@ -0,0 +1,11 @@
+{
+  "signers": {
+    "threshold": 0,
+    "weights": [
```

Comment:
It's odd that the `weights` need to be specified when we are trying to delete the list of signers.

Notice that in the rippled API, the SignerEntries field is omitted when deleting: https://xrpl.org/signerlistset.html

I think we can make the `weights` field optional: https://github.com/ripple/ripple-lib/blob/develop/src/common/schemas/objects/settings-plus-memos.json#L90

Then, it probably makes sense to add validation to ensure that if `threshold` is 0, then `weights` must not exist; if threshold >= 1, then `weights` is required.

Reply:
Thank you for your input. I agree with you on making the weights field optional based on the threshold value.